### ISSUE TITLE
Conversion impossible on Apple Silicon M1

### ISSUE THREAD

**Issue by zacksldr** [upscayl/custom-models#14]
Hi

I don't know if it is normal but conversion is not supported on M1. 

This is the error: 

Error

An error occurred in a PyTorch Convert To NCNN node:

DAT is not supported for NCNN conversion at this time.

Input values:
• PyTorch Model: Value of type 'nodes.impl.pytorch.architecture.DAT.DAT'
• Data Type: fp16

Thanks

**Comment by NayamAmarshe** [comment]
Yes, DAT architecture is not supported by Upscayl at the moment. Nor does Chainner support it.
